### PR TITLE
Bug/unity 1137 fixes for image alignment distortion issues

### PR DESCRIPTION
**Pull request by MaxPalmer-UH**
## Summary

This PR addresses issue UNITY-1137, where hands are misaligned with IR images in the control panel on certain devices. This is because the distortion matrix is treated differently between devices. The changes flip the Y value in the distortion matrix data, this also flips the IR image, so we invert the V value in the UV calculation in the shader code. 

This change makes things work in theory across device types (SIR170, 3Di, LMC). However, in fact this will then break LMC alignment. There is a new version of the service which changes the LMC distortion data to work with this fix.

This change backs out the left/right distortion matrix swap, which appeared to fix alignment across all devices but in fact was the wrong solution.

As this bug has taken a while to track down, I've added new comments to boost understanding of the 'rendering pipeline' for images.

Testing requirements:
Needs to be tested with a version of the control panel that uses a plugin with these changes. For IR/Hand alignment to work across all device types, requires a new version of the SDK/Service (5.12 RC4). Note, some devices will not obviously show misalignment so you need one that shows the issues before the fix. Because of the way SIR170s treat the image centre, they also don't show alignment issue.

## Contributor Tasks

_These tasks are for the merge request creator to tick off when creating a merge request._

- [ ] Pair review with a member of the QA team.
- [x] Add any release testing considerations to the MR for the next release.
- [x] Check any relevant CHANGELOG files have been updated.
- [x] Ensure documentation requirements are met e.g., public API is commented.
- [x] Consider any licensing/other legal implications for this MR e.g. notices required by any new libraries.
- [x] Add any relevant labels such as `breaking` to this MR.
- [ ] If this MR closes a Jira issue, make sure the fix version on the JIRA issue is set to the correct one.

## Reviewer Tasks

_Add any instructions or tasks for the reviewer such as specific test considerations before this can be merged._

[Use emojis in review threads to communicate intent and help contributors.](https://github.com/ultraleap/UnityPlugin/blob/develop/CONTRIBUTING.md#review-threads)

- [x] Code reviewed.
- [x] Non-code assets e.g. Unity assets/scenes reviewed.
- [ ] Documentation has been reviewed. Includes checking documentation requirements are met and not missing e.g., public API is commented.
- [ ] Checked and agree with release testing considerations added to MR for the next release.

## Closes JIRA Issue

_If this MR closes any JIRA issues list them below in the form `Closes PROJECT-#`_

### CODE REVIEW

File: Packages/Tracking/CHANGELOG.md [hunk 1]
```diff
@@ -16,6 +16,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 - Preview package version dependency mismatch for XRI when using InputSystem 1.4.4
+- Control panel IR image / hand vertical alignment across device types and individual devices, through changing the handdling of the distortion matrix
```
Comment (MattGrayUL) [inline Packages/Tracking/CHANGELOG.md:19]: Do we need to mention the control panel here? the changelog only really relates to the Plugin, so we could just mention the passthrough distortion matrix, but not the control panel?

**Review by MattGrayUL — APPROVED** [review]
Happy with the changes, tested in editor on the Control Panel and the alignment was resolved

Just a couple of comments about changelog and commenting

File: Packages/Tracking/CHANGELOG.md (hunk 1 above)
Comment (MaxPalmer-UH) [reply Packages/Tracking/CHANGELOG.md]: Good suggestion, changed.

File: Packages/Tracking/Core/Runtime/Plugins/LeapCSharp/Image.cs [hunk 2]
```diff
@@ -151,7 +151,7 @@ public float[] Distortion(CameraType camera)
             if (camera != CameraType.LEFT && camera != CameraType.RIGHT)
                 return null;
 
-            return imageData(camera).DistortionData.Data;
+            return imageData(camera).DistortionData.FlippedData;
```
[the remark this answers is not in the document]
Comment (MaxPalmer-UH) [reply Packages/Tracking/Core/Runtime/Plugins/LeapCSharp/Image.cs:155]: I've added a comment.

**Pull request merged**